Fix/html

novaraptur opened the pull request:
#### - [ ] Feature
#### - [X] Fix
#### - [X] Refactor
#### Comment:  Merged Andrew's API changes / fetch code; updated HTML & scripts to make code drier and more dynamic. Removed redundant sections & classes in CSS, added better accessibility features like semantic HTML.

Commit:
Fix spacing and indentation

Commit:
Change non-stylistic div cards to articles

Commit:
Remove unnecessary section and add explanatory comments to user sidebar

Commit:
Refactor card container styles in CSS

Commit:
Remove unused / redundant sidebar styling

Commit:
Remove redundant CSS for user name display

Commit:
Remove overlapping ids and references to now non-existent class

Commit:
Add further explanatory comments to HTML

Commit:
Add alt text to images and rename CSS class

Commit:
Remove / refactor redundant classes for user info in sidebars

Commit:
Fix formatting

Commit:
Remove references to non-existent CSS class

Commit:
Remove redundant / unnecessary CSS class

Commit:
Remove redundant / unnecessary CSS class

Commit:
Remove unused CSS class 'number'

Commit:
Restore number class

Commit:
Refactor activity and sleep card classes and remove references to non-existent class from scripts.js

Commit:
Remove empty section and fix ul formatting

Commit:
Remove comments

Commit:
Remove div elements and add border-bottom instead

Commit:
Reduce CSS class redundancy in activity/sleep/hydration cards

Commit:
Add hydration info to first hydration card dynamically through JS

Commit:
Add hydration info to second hydration card dynamically through JS

Commit:
Remove extra section

Commit:
Add activity info dynamically to first activity card through JS

Commit:
Add activity info dynamically to second activity card through JS

Commit:
Add sleep info dynamically to cards through JS

Commit:
Remove unnecessary query selected variables

Commit:
Add new JS file for DOM manipulation

Commit:
Prep for PR

Commit:
Merge branch 'main' into html fix branch and fix merge conflicts

Commit:
Fix image casing

andrewvallejo commented on dist/index.html:
```diff
@@ -1,201 +1,53 @@
-</body>
-
+    </main>
+    <script src="bundle.js"></script>
```
Wow, I knew there was extra fluff but you really thinned this down! I love the explicit tags too!

andrewvallejo committed:
Update scripts.js

I added these changes to better reflect our progress with the fetch API. You still need to fix some pathing issues...

andrewvallejo reviewed (approved):
I love all of these changes. Please refer to my other comments!
